### DISCUSSION TITLE
使用ilogtail1.7.1 docker版本发现占用的虚拟内存比较大

### DISCUSSION THREAD

**Discussion by 8wy162493** [alibaba/ilogtail#1168] [Help]
使用ilogtail1.7.1 的docker版本，在docker 启动后，虚拟内存会占用的比较大。企业版的占用的虚拟内存就小的很多。所以是否有版本能限制虚拟内存的大小。我已经使用--m 和 [--memory-swap以及--memory-swappiness都进行了限制。但是虚拟内存那块依然很大。所以求助有没有办法能限制。
`docker run -d --name docker_ilogtail \
  --memory-swappiness=2 \
  --network=host \
  -v /:/logtail_host:ro \
  -v /var/run:/var/run \
  -v /opt/starcharge/ilogtail_config.json:/usr/local/ilogtail/ilogtail_config.json \
  -v /var/lib/docker_ilogtail/checkpoint:/usr/local/ilogtail/checkpoint \
  -v /opt/xxxxxx/user_yaml_config.d:/usr/local/ilogtail/user_yaml_config.d \
  sls-opensource-registry.cn-shanghai.cr.aliyuncs.com/ilogtail-community-edition/ilogtail:1.7.1`

![iShot_2023-10-08_17 54 51](https://github.com/alibaba/ilogtail/assets/8033763/51e778d4-2d1d-4f18-9dc3-8be5a4dff10f)

**Comment by linrunqi08** [comment]
@8wy162493  商业版的占用是多少呢？另外两遍的采集配置量级以及采集的数据量是否一致呢
